src/components/SelectBox.js

```diff
@@ -43,16 +44,30 @@ class SelectBox extends React.Component {
       this.props.onChange(id)
     }
   }
+  getValidationState () {
+    const { validationState } = this.props
+    switch (validationState) {
+      case 'error':
+        return style['selectBox-has-error']
+      case 'warning':
+        return style['selectBox-has-warning']
```

Comment:
I think we don't need this case right now. I would avoid adding unnecessary code which can be added later anytime if needed. Otherwise, LGTM.

Reply:
Hi Hilda, I agree that is not necessary right now, but IMHO it's just component's functionality, so if we already edited the component, it will be better to add the functionality to create consistency with project's UI library - PF3 WDYT?

Reply:
I still think we should avoid adding unnecessary code. We don't know if we will use it in the future.